AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/config/NewConfig.java

```diff
@@ -318,6 +319,7 @@ public void loadDefaults() {
                 "For example, having homes A, B, C, D and E with a limit of 3 will deny access to D and E.");
         addDefault("hide-homes-if-denied", false, "If homes should be hidden from /homes should they be denied access.\n" +
                 "If this is false, they will be greyed out in the /homes list.");
+        addDefault("overwrite-sethome", false, "When enabled, setting homes with a name that already exists in your list gets overwritten.");
```

Comment:
With these sorts of stuff (see the postSave method, or line 564) you will need to declare OVERWRITE_SETHOME, unfortunately these variables don't set themselves 😩